### PR TITLE
Fix DT tables selected row highlight with updating css

### PR DESCRIPTION
**Pull request by esraay**
Short description (with a reference to an issue).

**DoD**

- [ ] Major project work has a corresponding task. If there’s no task for what you are doing, create it. Each task needs to be well defined and described.

- [ ] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.

- [ ] No new error or warning messages are introduced.

- [ ] All interaction with a semantic functions, examples and docs are written from the perspective of the person using or receiving it. They are understandable and helpful to this person.

- [ ] If the change affects code or repo sctructure, README, documentation and code comments should be updated. 

- [ ] All code has been peer-reviewed before merging into any main branch.

- [ ] All changes have been merged into the main branch we use for development (develop).

- [ ] Continuous integration checks (linter, unit tests) are configured and passed.

- [ ] Unit tests added for all new or changed logic.

- [ ] All task requirements satisfied. The reviewer is responsible to verify each aspect of the task.

- [ ] Any added or touched code follows our style-guide.

### CODE REVIEW

**Review by jakubsob — COMMENTED** [review]
What about the other part of the issue?
> Also, when trying to change the table style (e.g. by adding style = 'default') , an error message appears:
Error: formal argument "style" matched by multiple actual arguments

We could create a new issue for fixing the `style` argument

File: inst/www/shiny-semantic-DT.css [hunk 1]
```diff
@@ -16,3 +16,6 @@
   -webkit-transition: color .1s ease,border-color .1s ease;
   transition: color .1s ease,border-color .1s ease;
 }
+table.dataTable tr.selected td, table.dataTable td.selected {
+  background-color: #95d8f5 !important;
```
[the remark this answers is not in the document]
Comment (esraay) [reply inst/www/shiny-semantic-DT.css]: Thanks for noticing, yes it was unnecessary so I removed it.

**Comment by esraay** [comment]
> What about the other part of the issue?
> 
> > Also, when trying to change the table style (e.g. by adding style = 'default') , an error message appears:
> > Error: formal argument "style" matched by multiple actual arguments
> 
> We could create a new issue for fixing the `style` argument

'style' argument added to semantic_DT

**Comment by esraay** [comment]
Also, I added the class argument for semantic_DT after checking out #417. In this way, all classes provided in fomanticUI official documentation and DT's default classes can be used.

**Review by jakubnowicki — APPROVED** [review]

**Pull request merged**